<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员登录</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QLabel" name="label_80">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入管理员密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>81</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>81</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 0);
    color: black;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
